File: backend/Qubik.Hackathon.API/Models/Startup.cs
namespace Qubik.Hackathon.API.Models
{
    public class Startup
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public ICollection<Report> Reports { get; set; }
        public ICollection<Milestone> Milestones { get; set; }
        public ICollection<Investment> Investments { get; set; }
    }
}

File: frontend/types/api/Startup.ts
import type { Milestone } from "./Milestone";
import type { Transaction } from "./Transaction";

export interface Startup {
    id: number;
    name: string;
    address: string;
    imageUrl: string;
    reports: any[] | null;
    investorIdentity: string;
    investorSeed: string;
    milestones: Milestone[];
    investments: any[];
    transactions: Transaction[];
}
